### ISSUE TITLE
Confusing Error Message when Using Multiple Form Validators

### ISSUE THREAD

**Issue by markheron91** [angular/angular#13951]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
```

**Current behavior**

When using reactive forms and adding multiple validators to a form field, there is a confusing error message that occurs if you accidentally leave out the array brackets around the list of validators. The error message contains "cannot read 'subscribe' of null" and the stack trace has form validation related things in it, but it is not clear that this is the error. I realise that this is not a huge problem given that the stack trace contains some information on where to start looking for the error but it might be good (if possible) to give a clearer picture to the developer what has gone wrong in this case.

**Expected behavior**

The error message produced when trying to add multiple validators to a form control without adding them as an array could read something like 'FormBuilder expected 2 args ( value: any, validators: any[]) but received n'

**Minimal reproduction of the problem with instructions**

https://plnkr.co/edit/Jab1AKSCMco2lQZwhtiF?p=preview

To see the problem remove the brackets from around the validators list and view the error in the plunkr

**What is the motivation / use case for changing the behavior?**

Small error cost a lot of time to debug.

**Please tell us about your environment:**

Windows 10 home edition, vscode insiders

* **Angular version:** 2.0.X

2.4.3 (router 3.4.3)

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

All
 
* **Language:** [all | TypeScript X.X | ES6/7 | ES5]
Typescript 2.1.5

* **Node (for AoT issues):** `node --version` =

**Comment by DzmitryShylovich** [comment]
plunkr doesn't load

**Comment by markheron91** [comment]
think I had it set as a private plunkr - forked it to a public one here https://plnkr.co/edit/4sWCQno5pB6YB4089Xf2?p=preview

**Comment by DzmitryShylovich** [comment]
I don't see any errors. How to reproduce?

**Comment by markheron91** [comment]
@DzmitryShylovich in the app.component.ts file remove the square brackets around the multiple validators for the form field - it should look like units: [ '', Validators.required, unitsValidator ] and once it refreshes the error will show up

**Comment by Toxicable** [comment]
It also gives a TypeScript Error which I think should be enough to point your attention towards your syntactical error.
However, this is still a point where the Error messages could be improved.
FormBuilder.group() for ref
https://github.com/angular/angular/blob/14ee75924b6ae770115f7f260d720efa8bfb576a/modules/%40angular/forms/src/form_builder.ts#L44

**Comment by DzmitryShylovich** [comment]
just need to verify that async validator is actually an observable and throw a meaningful error.

**Comment by pkozlowski-opensource** [comment]
Fixed by 94f84c5
